### PR TITLE
fixed deprecation msg and added an UPGRADE note

### PR DESCRIPTION
**Pull request by OskarStark**
I am targetting this branch, because this is BC.

Refs #1124

## Subject

- Fixed the wrong deprecation message, which was introduced here: https://github.com/sonata-project/SonataMediaBundle/pull/1124#discussion_r97044541 and added an UPGRADE note
- added to two more deprecations in separate commits

### CODE REVIEW

File: UPGRADE-3.x.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 UPGRADE 3.x
 ===========
 
+### Deprecated to inject 4th parameter in Sonata\MediaBundle\Thumbnail\ConsumerThumbail::__construct() optional
```
Comment (OskarStark) [inline UPGRADE-3.x.md]: this sounds weird, any idea @greg0ire ?

Comment (greg0ire) [inline UPGRADE-3.x.md]: Not providing the 4th argument to blah blah blah is deprecated

Comment (OskarStark) [inline UPGRADE-3.x.md]: thx

**Comment by OskarStark** [comment]
added 2 more @greg0ire

**Comment by OskarStark** [comment]
Thank you guys 👍

**Pull request merged**